style/global.scss

```diff
@@ -82,3 +82,9 @@ h6 {
     flex-direction: column;
   }
 }
+
+nav * {
+  font-familY: var(--font-family-primary) !important;
+  font-weight: 700;
```

Comment:
Is there a reason to use `700` rather than just `bold`? 

I notice elsewhere in the child theme we've used `bold` and they seem to be the [same](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight) anyway.

Reply:
I just used 700 because I'm used to the standard being to use numbers and I didn't really think about it too hard. Happy to change if we are using a different standard.